fe/fe-core/src/main/java/com/starrocks/lake/LakeTablet.java

```diff
@@ -58,6 +58,8 @@ public class LakeTablet extends Tablet {
     private long dataSize = 0L;
     @SerializedName(value = JSON_KEY_ROW_COUNT)
     private long rowCount = 0L;
+    @SerializedName(value = "maxModificationTime")
+    private long maxModificationTime = 0L;
```

Comment:
The name `maxModificationTime` is a bit too broad, requiring more context to understand its meaning. How about `lastStatisticsUpdateTime`?